schema/docs/elements_o2r.md

```diff
@@ -17,11 +17,11 @@ _Last updated: 2017-12-01, work in progress!_
 - `depends.operatingSystem` _String_.
 - `depends.identifier` _String_.
 - `depends.packageSystem` _String_. URL
-- `depends.version` _String_.
+- ~~`depends.version` _String_~~ _Deprecated_.
 - `description` _String_. A text representation conveying the purpose and scope of the asset (the abstract).
 - `displayfile` _String_. The suggested file for viewing the text of the workspace, i.e. a rendering of the suggested mainfile.
 - `displayfile_candidates` _Array of strings_. An unsorted list of candidates for displayfiles.
-- `ercIdentifier` _String_. A universally unique character string associated with the asset as executable research compendium, provided by the o2r service.
+- ~~`ercIdentifier` _String_. A universally unique character string associated with the asset as executable research compendium, provided by the o2r service.~~ _Deprecated_.
```

Comment:
Suggest to not have "deprecated" stuff at all. We haven't made a proper "release" yet (meaning that we bugfix older versions) nor do we plan to, so simply really remove unsupported fields.

Reply:
I agree. I did this to support our team to whom the whereabouts of that element might be puzzling. In fact a lot of issue in this repository are the result of similar events.

Reply:
Right. But the current state is that only I will have to remember :-)

So a reasonable practice to me would be to remove each deprecated feature in its own commit and mention it in the commit message, this way the change is findable.

Reply:
Okay